github/filecoin-project.yml

```diff
@@ -3946,59 +3848,32 @@ repositories:
         - proofs-crate-owners
       maintain:
```

Comment:
On all other repos, `proofs` has only push access. I don't think `merkletree` should be a special case here.

Reply:
Addressed in https://github.com/filecoin-project/github-mgmt/pull/133/commits/52590c1c21dff9a0865b8029b12d861aa277f5da